appservice/src/main/resources/static/swagger/swagger-spec.yaml

```diff
@@ -1892,28 +1880,32 @@ definitions:
     - ANDROID
     - IOS
     - UNKNOWN
+    example: ANDROID
+  DeviceAppVersion:
+    type: string
+    description: The version of the Yona app running on this device. This is the marketing version, a free text string.
+    example: 1.1 build 83
+  DeviceAppVersionCode:
+    type: integer
+    description: The version code of the Yona app running on this device. This is positive a monotonic increassing number, across releases
```

Comment:
`positive a monotonic increasing number` -> `a positive, monotonic increasing number`

Reply:
Done

Reply:
Corrected this line in #494